udata/core/site/models.py

```diff
@@ -11,8 +16,8 @@
 
 
 class SiteSettings(db.EmbeddedDocument):
-    home_datasets = db.ListField(db.ReferenceField('Dataset'))
-    home_reuses = db.ListField(db.ReferenceField('Reuse'))
+    home_datasets = db.ListField(db.ReferenceField(Dataset))
+    home_reuses = db.ListField(db.ReferenceField(Reuse))
```

Comment:
I would have imagined a method for this class that would filter out deleted datasets.